facia/app/controllers/FaciaController.scala

```diff
@@ -97,14 +95,13 @@ trait FaciaController extends Controller with Logging with ExecutionContexts wit
 
   def redirectTo(path: String)(implicit request: RequestHeader): Future[Result] = successful {
     val params = request.rawQueryStringOption.map(q => s"?$q").getOrElse("")
-    Cached(60)(WithoutRevalidationResult(Found(LinkTo(s"/$path$params"))))
+    Cached(CacheTime.Facia)(WithoutRevalidationResult(Found(LinkTo(s"/$path$params"))))
```

Comment:
although in reality, redirects aren't likely to change often so could be cached much longer - facia (and article) already return an excessive number of redirects